torch/nn/utils/clip_grad.py

```diff
@@ -33,6 +35,19 @@ def clip_grad_norm_(parameters, max_norm, norm_type=2):
     return total_norm
 
 
+def clip_grad_norm(parameters, max_norm, norm_type=2):
+    r"""Clips gradient norm of an iterable of parameters.
+
+    .. warning::
+        This method is now deprecated in favor of
+        :func:`torch.nn.utils.clip_grad_norm_`.
+    """
+    warnings.warn("torch.nn.utils.clip_grad_norm is now deprecated in favor "
+                  "of torch.nn.utils.clip_grad_norm_.",
+                  category=DeprecationWarning, stacklevel=2)
```

Comment:
Let's not set it as a `DeprecationWarning`. `DeprecationWarning`s are ignored by default.